src/components/CopyButton/CopyButton.tsx

```diff
@@ -7,14 +7,7 @@ import { ButtonProps, Button } from "../Button/Button"
 import { ThemeCss } from "../../theme"
 
 const baseCss: ThemeCss = _theme => ({
-  "html:not([dir=rtl]) &": {
-    borderTopLeftRadius: 0,
-    borderBottomLeftRadius: 0,
-  },
-  "html[dir=rtl] &": {
-    borderTopRightRadius: 0,
-    borderBottomRightRadius: 0,
-  },
+  fontWeight: 600,
```

Comment:
```suggestion
  fontWeight: _theme.fontWeights.semibold,
```

(probably wanna drop the leading underscore)